test/hello_world.2asm.js

```diff
@@ -13,8 +13,8 @@ function asmFunc(global, env, buffer) {
  var Math_abs = global.Math.abs;
  var Math_clz32 = global.Math.clz32;
  function add(x, y) {
-  x = x | 0;
-  y = y | 0;
+  x = (x | 0)
```

Comment:
It's safer to leave extra semicolons in (no semicolons can run into automatic semicolon insertion nonsense in js...). Let's do that, and maybe figure out how to remove then later.